Map/Include/Game.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
```

Comment:
I brakuje nam jeszcze klasy testowej GameTest. Powinnien być w niej jeden test, który sprawdzi czy wszystko buduje się poprawienie

TEST_F(xxx, yyy)
{
ASSERT_TRUE(true);
}

Oczywisicie pod xxx i yyy odpowiednie nazwy. 


Jak nie pamiętasz jak pisać TEST_F, to możesz zobaczyć jak to było zrobione w pliku MapTest.cpp